[2021-10-12 13:42:21,332] {taskinstance.py:897} INFO - Dependencies not met for <TaskInstance: estate_pipeline.data_reader 2021-10-07T00:00:00+00:00 [up_for_retry]>, dependency 'Not In Retry Period' FAILED: Task is not ready for retry yet but will be retried automatically. Current date is 2021-10-12T05:42:21.332395+00:00 and task will be retried at 2021-10-12T05:44:36.636056+00:00.
[2021-10-12 13:42:21,335] {local_task_job.py:96} INFO - Task is not able to be run
